src/app/directives/documentation.tpl.html

```diff
@@ -186,6 +186,33 @@ <h4 class="raml-console-resource-body-heading">
             <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[code][responseInfo[code].currentType].example)"></code></pre>
           </div>
 
+          <div ng-if="responseInfo[code][responseInfo[code].currentType].examples">
+            <span>Examples:</span>
+            <div ng-if="isArray(responseInfo[code][responseInfo[code].currentType].examples)">
```

Comment:
We can do that, but we will lose the example name. For example, with this RAML:
```
/resource:
  get:
    responses:
      200:
        body:
          application/json:
            examples:
              one:
                content: |
                  {
                    "property1": "value1"
                  }
              two:
                content: |
                  {
                    "property2": "value2"
                  }
```

the use of 'one' and 'two' to distinguish the examples will be lost in the console.